sdk/storage/storage-blob/src/models.ts

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-
 import * as Models from "./generated/src/models";
 
+
 export interface Metadata {
```

Comment:
Remove line above.

Reply:
It should be prettier's default format to add additional line between import with below codes. Will check.

---
In reply to: [318949630](https://github.com/Azure/azure-sdk-for-js/pull/4874#discussion_r318949630) [](ancestors = 318949630)

Reply:
Fixed

---
In reply to: [319813499](https://github.com/Azure/azure-sdk-for-js/pull/4874#discussion_r319813499) [](ancestors = 319813499,318949630)